src/AssemblyPath.cs

```diff
@@ -0,0 +1,29 @@
+/*----------------------------------------------------------------
+ *  Copyright (c) ThoughtWorks, Inc.
+ *  Licensed under the Apache License, Version 2.0
+ *  See LICENSE.txt in the project root for license information.
+ *----------------------------------------------------------------*/
+
+
+namespace Gauge.Dotnet
+{
+    public class AssemblyPath
```

Comment:
Why do we need this?

Reply:
this is because gauge-dotnet now uses aspnet core's DI, and it's not easy to inject a dependency with primitive types.

Specifically, this lets us [do this](https://github.com/getgauge/gauge-dotnet/pull/123/files#diff-280ecf7c70e274bad023c9acea61103fae54640ec4fdc58b35dd6ff40d199e74R52)